scss/components/_inline-datepicker.scss

```diff
@@ -0,0 +1,49 @@
+$form-control-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+$inline-error-bg: #fbe3e4;
+
+.civicase-inline-datepicker__wrapper {
+  [civicase-inline-datepicker] {
+    border: 1px solid transparent;
+    border-radius: $input-border-radius;
+    height: $input-height-base;
+    padding: $padding-base-vertical $padding-base-horizontal;
+    width: 140px !important;
+  }
+
+  .form-control {
+    box-shadow: none;
+    display: inline-block;
+    margin-left: -($padding-base-horizontal + 1px);
```

Comment:
Why +1  is needed?

Reply:
Changed the code a bit here. The goal is to get negative margins to position the input element and the date text inside the input. The text should stay in the same position as it would have been if it were not an inline datepicker, and the input should move out respecting the paddings of the input and its borders.

1px is a constant because there's no variable for the input's border:
https://github.com/civicrm/org.civicrm.shoreditch/blob/master/base/scss/vendor/bootstrap/_forms.scss#L143